### ISSUE TITLE
[bitnami/redis] Can't open or create append-only dir appendonlydir: Permission denied

### ISSUE THREAD

**Issue by yangshifei** [bitnami/charts#14327]
### Name and Version

bitnami/charts/redis

### What steps will reproduce the bug?
```
Can't open or create append-only dir appendonlydir: Permission denied
```

### What do you see instead?

Can't open or create append-only dir appendonlydir: Permission denied

**Comment by javsalgar** [comment]
Hi,

Could you add more details of the Kubernetes cluster you are using? It seems to me the issue lies in the volume provisioning technology, that may not be compatible with the non-root configuration of the chart.

**Comment by kisdaniel** [comment]
I have a same issue with my v1.23.17 arm kubernetes cluster.

I installad it with kustomize:

`
helmCharts:
  - name: redis
    namespace: my-redis-cluster
    repo: https://charts.bitnami.com/bitnami
    valuesFile: redis-values.yaml
`

redis-values.yaml:
`
master:
  password: "*****"
persistence:
  enabled: true`


conainer output error message:
`
19 Jul 2023 10:15:35.677 # oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
19 Jul 2023 10:15:35.677 # Redis version=7.0.12, bits=64, commit=00000000, modified=0, pid=1, just started
19 Jul 2023 10:15:35.677 # Configuration loaded
19 Jul 2023 10:15:35.679 * monotonic clock: POSIX clock_gettime
19 Jul 2023 10:15:35.681 * Running mode=standalone, port=6379.
19 Jul 2023 10:15:35.681 # Server initialized
9 Jul 2023 10:15:35.687 # Can't open or create append-only dir appendonlydir: Permission denied
`

**Comment by ElioLopez** [comment]
just in case someone stumble with this one, setting this values solved the problem for me: in chart 18.1.0:
`volumePermissions:
    enabled: true
`
